src/main/java/discord4j/discordjson/json/MessageCreateRequest.java

```diff
@@ -14,6 +15,10 @@ static ImmutableMessageCreateRequest.Builder builder() {
     }
 
     Possible<Object> nonce();
+
+    @JsonProperty("enforce_nonce")
+    Possible<Boolean> enforceNonce();
```

Comment:
why here and not in the MessageSendRequestBase? or is for the nonce field?

Reply:
Because it is not supported for webhooks (which also extends MessageSendRequestBase)